bats/compatibility/runner.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+if [[ $(git diff --stat) != '' ]]; then
```

Comment:
Might move this block down below the functions to the main execution body of the script.